src/scripts/announce-restart.ts

```diff
@@ -36,6 +36,10 @@ async function abortRestart(): Promise<void> {
     );
 }
 
+const delay = (time: number): Promise<void> =>
```

Comment:
We already have a delay function, do we need this here as well

Reply:
Importing anything from `utils` has a dependency on our internal logger. Our internal logger has issues when writing to logs both from Docker and non-Docker scenarios, because Docker writes to the file as root. Would rather just avoid that for now and fix it later